openlibrary/plugins/worksearch/subjects.py

```diff
@@ -60,7 +60,8 @@ def GET(self, key):
             'lending_edition_s': '*'
         })
 
-        subj.v2 = True
+        if subj:
+            subj.v2 = True
         delegate.context.setdefault('bodyid', 'subject')
```

Comment:
I think this line can be dropped? Aren't all pages v2 now?

I thought this was just temporary while we migrated pages to responsive mode.

Reply:
Why is 'get_subject()` failing to find https://openlibrary.orgsubjects/fantasy ?  Is that the filter values?  What about that TODO?

Reply:
I will create an issue